src/bin/inx-chronicle/cli/mod.rs

```diff
@@ -2,6 +2,10 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use chronicle::db::mongodb::config as mongodb;
+#[cfg(feature = "inx")]
```

Comment:
Can we just use the full paths below

Reply:
I felt exactly the same so I did this without even reading your comment. I think I have a small Alex Coats in my head.